src/simulate.jl

```diff
@@ -20,8 +20,12 @@ all the corresponding data.
 
 `parameters` contains all the simulation information, including the number of
 replications, the type of simulation, the hydrological years to sample from, etc.
+
+### Keyword Arguments
+`async` is a boolean that sets whether SDDP.jl will run in parallel. Currently
+recommended to be set to false.
 """
-function simulate(JADEmodel::JADEModel, parameters::JADESimulation)
+function simulate(JADEmodel::JADEModel, parameters::JADESimulation; async::Bool=false)
```

Comment:
Bikeshedding.

I don't particularly like `async` because people might get confused with `SDDP.Asynchronous` which shouldn't be used.

What about:

 * `parallel_scheme::SDDP.AbstractParallelScheme = SDDP.Serial()`? I guess this forces the user do do `using SDDP` and `parallel_scheme = SDDP.Threaded()`
 * `parallel::Bool = false`
 * `multithreaded::Bool = false`

Reply:
> Just left it because it was what was being used

Where?